### DISCUSSION TITLE
Including an extra file for deployments that's not in the repo

### DISCUSSION THREAD

**Discussion by Wolfyxon** [vercel/community#7447] [Help] [answered]
### Summary

Is it possible to upload an extra file to the deployments that's not included in the git repository where the deployments are imported from? I need that for a config file and I think including personal config files in the source code is a bad practice.

### Steps to Reproduce

N/A

**Comment by amyegan** [comment, marked as answer]
Hey @Wolfyxon! You could include the config value in an [Environment Variable](https://vercel.com/docs/projects/environment-variables#environment-variables) in **Project Settings** and load the value from that

**Reply by Wolfyxon** [reply]
It's not exactly what I wanted but it will do the job. Thanks.

[a comment by an automated account is not shown]
